### PR TITLE
Small updates to the project so it builds after a fresh clone of the repo.

### PR DESCRIPTION
**Pull request by robbforce**

### CODE REVIEW

**Comment by SanSYS** [comment]
> C:\Windows\System32\

System may be located in another disk, I recommend to use path like this: %WINDIR%\system32\

File: Bonobo.Git.Server/Bonobo.Git.Server.csproj [hunk 1]
```diff
@@ -654,9 +659,9 @@
     <PreBuildEvent>
 if not exist "$(TargetDir)NativeBinaries" md "$(TargetDir)NativeBinaries"
 if not exist "$(TargetDir)NativeBinaries\x86" md "$(TargetDir)NativeBinaries\x86"
-xcopy /s /y /d "$(SolutionDir)packages\LibGit2Sharp.0.18.1.0\lib\net40\NativeBinaries\x86\*.*" "$(TargetDir)NativeBinaries\x86"
+C:\Windows\System32\xcopy /s /y /d "$(SolutionDir)packages\LibGit2Sharp.0.18.1.0\lib\net40\NativeBinaries\x86\*.*" "$(TargetDir)NativeBinaries\x86"
```
Comment (SanSYS) [inline Bonobo.Git.Server/Bonobo.Git.Server.csproj]: > C:\Windows\System32\

System may be located in another disk, I recommend to use path like this: %WINDIR%\system32\

if it's possible, of course :)

Comment (SanSYS) [inline Bonobo.Git.Server/Bonobo.Git.Server.csproj]: > C:\Windows\System32\

System may be located in another disk, I recommend to use path like this: %WINDIR%\system32\

if it's possible, of course :)

**Comment by padremortius** [comment]
Environment variable **PATH** always contains the path %WINDIR%\system32\

**Comment by jakubgarfield** [comment]
Please update the xcopy as per SanSYS suggestion before merging.

**Comment by jakubgarfield** [comment]
So there is no need to prefix?

**Comment by padremortius** [comment]
Full path to xcopy not need.

**Comment by robbforce** [comment]
Like I mentioned in the commit, visual studio on my system (x64) exits before it copies the files, until I added the path to the xcopy command. I've updated the path with the %windir% variable now.

**Pull request merged**